Frontend/ui-driver/src/Screens/ReferralScreen/View.purs

```diff
@@ -1169,3 +1172,9 @@ emptyView =
   [ width WRAP_CONTENT
   , height WRAP_CONTENT
   ][]
+
+checkDriverWithZeroRides :: ST.RankCardData -> Boolean -> ST.ReferralScreenState -> Boolean
+checkDriverWithZeroRides item aboveThreshold state = 
+  let currentDriverData = state.props.currentDriverData
+  in  if aboveThreshold && (item == currentDriverData && currentDriverData.rank == 0) then true 
```

Comment:
instead of using this as a condition and sending it as true or false. we can directly send the condition

Reply:
Done